src/Microsoft.TestPlatform.ObjectModel/Microsoft.TestPlatform.ObjectModel.csproj

```diff
@@ -1,13 +1,14 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
+    <OverrideImplicitNSVersion Condition=" '$(TargetFramework)' == 'netstandard2.0' ">true</OverrideImplicitNSVersion>
     <TestPlatformRoot Condition="$(TestPlatformRoot) == ''">..\..\</TestPlatformRoot>
   </PropertyGroup>
   <Import Project="$(TestPlatformRoot)scripts/build/TestPlatform.Settings.targets" />
   <PropertyGroup>
     <AssemblyName>Microsoft.VisualStudio.TestPlatform.ObjectModel</AssemblyName>
-    <TargetFrameworks>net451;netstandard1.4</TargetFrameworks>
-    <TargetFrameworks Condition=" '$(DotNetBuildFromSource)' == 'true' ">netstandard1.4</TargetFrameworks>
+    <TargetFrameworks>net451;netstandard1.4;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks Condition=" '$(DotNetBuildFromSource)' == 'true' ">netstandard2.0</TargetFrameworks>
```

Comment:
why is it changes only for one project? Also I don't get the need to upgrade it to 2.0 when NS1.4 will work? Is it because of below dependency of "System.Runtime.InteropServices.RuntimeInformation"

Reply:
The reason for that is that netstandard < 2.0 brings framework assemblies with it which results in them being copied to the output directory. With >= netstandard2.0 we changed that so that nothing will be brought and copied over. We always offer both the minimum possible NS version + a NS2.0 version for our packages in .NET Core. Upgrading both the ObjectModel package and using a newer version of the DependencyModel package results in a clean output graph and we can finally use it in CoreFx.

As we produce the framework assemblies ourselves we would hit conflicts if Microsoft.NET.Test.Sdk brings their own with them. The remaining issue is Newtonsoft.Json of course but I think there is a workaround on our side for that.

Reply:
I don't think we need this `DotNetBuildFromSource` conditional override. Build from source can build any TFM.